static/app/views/issueList/groupSearchViewTabs/editableTabTitle.tsx

```diff
@@ -33,8 +31,6 @@ function EditableTabTitle({
   const inputRef = useRef<HTMLInputElement>(null);
   const isEmpty = !inputValue.trim();
 
-  const isSelected = tabListState?.selectedKey === tabKey;
```

Comment:
This is the line that caused the regression - the tabListState takes a second after initial page load to initiate, and when it does it bolds the tab title, causing a visual animation